client/src/pages/CreateUnit.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import Drawer from 'components/Drawer'
+import { colors, fonts } from 'theme'
+import {
+  Input,
+  Text,
+  TextArea,
+  View,
+  Keyboard,
+  TouchableWithoutFeedback,
+} from 'native-base'
+import { Foundation } from '@expo/vector-icons'
+import dismissKeyboard from 'react-native-dismiss-keyboard'
+
+const CreateUnit = ({ navigation }) => {
+  const close = () => {
+    navigation.goBack()
+  }
+
+  const success = () => {
+    // call api request here
+    console.log('success')
+  }
+
+  const [name, setName] = useState('')
+  const [purpose, setPurpose] = useState('')
+
+  const body = (
+    <>
```

Comment:
Create a 
```
const styles = StyleSheet.create({

})
``` 
at the top of the page, and add all of the styles that you use below to that. That way, the JSX looks a lot cleaner and there is a centralized location for all styling applied to this component. See `StyledCard` for an example.